notebooks/en/enterprise_dedicated_endpoints.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<p>"...the TGI container running on the endpoint applies..."</p><p> </p><p>"Here is the slightly modified code..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>